tests/Dfc.CourseDirectory.Testing/Dfc.CourseDirectory.Testing.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
+    <UserSecretsId>1a0e7451-56d9-4d15-a253-1d2bd6985264</UserSecretsId>
```

Comment:
We've used friendly names rather than GUIDs everywhere else for user secrets, I think.

Reply:
What advantage does that give over the default GUID? Do you need to deal with them manually?